jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -283,11 +285,21 @@ YUI.add('user-profile', function() {
     */
     _generateModelRow: function(model) {
       var uuid = model.uuid;
+      var isCurrent = model.name === this.props.currentModel;
+      if (!model.isAlive) {
```

Comment:
OIC so we ni longer display the component at all if the model is not alive, so ignore my previous comments, and yay github reviews live comments!